### ISSUE TITLE
[RAI] July Core Book, End of Next Turn Resolution

### ISSUE THREAD

**Issue by lenaleciel** [AshleyMoni/Lancer-Community-Edition#944]
**The Problem:**
There is a bit of ambiguity as to when effects that last until the end of a character's next turn actually end.
In simpler terms, it isn't particularly defined as to whether these effects are still ongoing when things that trigger at the end of the turn trigger, such as Burn, the Raleigh's Full Metal Jacket, the Autogun, and the Balor's Regeneration.

In other words, if "This effect that ends at the end of this turn" is itself "an effect that needs to resolve," then it falls under the Action resolution section on page 68, meaning effects caused by other characters trigger first, and otherwise, you are allowed to pick the resolution order of effects on your turn.

If that is how it is supposed to work, then this leads to behavior such as:
* Burn not being affected by Impair that lasts until the end of your next turn, because "Impair falls off" can resolve before "Burn's damage check applies". (Tom has said that it doesn't work this way before.)
* Stun not affecting the Raleigh's Full Metal Jacket or the Autogun: Both these effects trigger "at the end of your turn," which would resolve after the Stun resolves. (Other conditions and statuses can be affected by this in similar ways)
* The Balor's Regeneration shutting off "until the end of its next turn" doesn't actually do anything unless it gets structure or stress on its own turn, because you'd resolve the Regeneration effect after the "the Regeneration turns back on" effect.

That last one is particularly stupid, but it has genuinely come up in Rules Discussion before, repeatedly. 
 
However, all of these problems go away if effects that last "until the end of their next turn" or "until the end of the turn" or similar don't resolve or trigger, and literally just last until the turn is officially over.

**The Solution:**
Tom has previously stated that he doesn't want to give Lancer a whole MTG style resolution system, so this probably isn't much of a priority, but bullet point 2 up there has become common knowledge on Pilot.Net as how it works, and it would be good to check if that is how it is genuinely intended to work.

I've talked to Kai Tave about bullet points 2 and 3 up there, and he came to the conclusion that he doesn't know that if "until end of (next) turn" effects fall off before end of turn free actions is unintended or not, but if they are supposed to be triggered effects that resolve, you can just fix the Balor with sticking the word "Instead" on the end of the effect, to make it more clear that it replaces the regeneration effect.

**The Location:**
The Action Resolution rules are on page 68.
Balor is on Page 192.

**Comment by KaiTave** [comment]
The thing with burn and impair is that they're both probably coming from hostile sources, and that being the case they're going to resolve in the order that the GM chooses, not as a player decision.
